scripts/zones/Inner_Horutoto_Ruins/Zone.lua

```diff
@@ -129,6 +129,20 @@ zoneObject.onTriggerAreaLeave = function(player, triggerArea)
     end
 end
 
+zoneObject.onGameHour = function()
+    local totd = VanadielTOTD()
+    if totd == xi.time.NEW_DAY then
+        local bonesOne = GetMobByID(17563670)
+        local bonesTwo = GetMobByID(17563671)
```

Comment:
Might be better to add values to the zone's `mob` table so you can reference via ID.Mob.MAGICKED_BONES[1] and [2] or just _1 and _2

Reply:
These are references to the mob itself rather than just the id, it didn't look like there were other actual mobs in ID

Reply:
yep, you gotta add them!